script.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars */
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+const addBook = document.getElementById('book');
+
+const dataMod = [];
+
+function addNewBook() {
+  const book = {};
+  const rmv = 'Rmv';
+  book.title = titleInput.value;
+  book.author = authorInput.value;
+  dataMod.push(book);
+
+  const stringVal = titleInput.value;
+  const idString = stringVal.replace(/\s/g, '');
+  const rmvBtn = idString + rmv;
+
+  window.localStorage.setItem(idString, JSON.stringify(book));
```

Comment:
- [ ] It would be better if we fix the data persistence, here is what I mean. Once I entered several data in the UI, we can see some data in the local storage but once we refresh the page, all data is lost in the UI. Let us kindly investigate this :man_technologist: